diff --git a/node_modules/@tldraw/editor/dist-cjs/lib/TldrawEditor.js b/node_modules/@tldraw/editor/dist-cjs/lib/TldrawEditor.js
index cb45b44..2e10663 100644
--- a/node_modules/@tldraw/editor/dist-cjs/lib/TldrawEditor.js
+++ b/node_modules/@tldraw/editor/dist-cjs/lib/TldrawEditor.js
@@ -149,8 +149,7 @@ function TldrawEditorWithReadyStore({
       getContainer: () => container,
       user
     });
-    window.app = editor2;
-    window.editor = editor2;
+    window.tldraw = { app: editor2, editor: editor2 };
     setEditor(editor2);
     return () => {
       editor2.dispose();
diff --git a/node_modules/@tldraw/editor/dist-esm/lib/TldrawEditor.mjs b/node_modules/@tldraw/editor/dist-esm/lib/TldrawEditor.mjs
index c7140e1..7a6dbf1 100644
--- a/node_modules/@tldraw/editor/dist-esm/lib/TldrawEditor.mjs
+++ b/node_modules/@tldraw/editor/dist-esm/lib/TldrawEditor.mjs
@@ -126,8 +126,7 @@ function TldrawEditorWithReadyStore({
       getContainer: () => container,
       user
     });
-    window.app = editor2;
-    window.editor = editor2;
+    window.tldraw = { app: editor2, editor: editor2 };
     setEditor(editor2);
     return () => {
       editor2.dispose();
